crates/astria-composer/src/executor/mod.rs

```diff
@@ -199,6 +203,16 @@ impl Executor {
     /// An error is returned if connecting to the sequencer fails.
     #[instrument(skip_all, fields(address = %self.address))]
     pub(super) async fn run_until_stopped(mut self) -> eyre::Result<()> {
+        let remote_chain_id = self
+            .get_sequencer_chain_id()
+            .await
+            .wrap_err("failed obtain sequencer chain_id")?;
+        ensure!(
+            self.sequencer_chain_id == remote_chain_id,
+            "mismatch in configured chain_id: config specifies {0}, but sequencer rpc is for {1}",
```

Comment:
You don't need the numbers, and you can rely on interpolation (also I recommend backticks to make it easier for a human to parse the text). I also suggest to remove mention of "config" - it's not not wrong, but this part of the code has no concept of the actual config, so I'd leave it more generic:
```suggestion
            "mismatch in Sequencer chain IDs: expected `{}`, but Sequencer returned `{remote_chain_id}`",
            self.sequencer_chain_id,
```

Reply:
Resolved in 8523e04.